.github/ISSUE_TEMPLATE/-onboarding--welcome-a-new-team-member.md

```diff
@@ -7,51 +7,57 @@ assignees: ''
 
 ---
 
-# Description 
+Read through these links and check off items.
 
-Here are some tasks to go through that should help you get to know the team and product better to be able collaborate with the team.
-
-Present checklist is here to help us track progress and completion of this.
+# Readings
+## Learn the product
+- [ ] [Overview of Notify product](https://github.com/cds-snc/notification-planning/wiki/Overview) 
+- [ ] [Notify Scavenger Hunt](https://docs.google.com/document/d/1MmKBZqMNpsLU0TlURGCyOlByZHGfZccteuYdGZ9amwM/edit#heading=h.sxqbcn4k0lmo)🕵️‍♀️
+- [ ] [Team OKRs 2023-2024](https://docs.google.com/document/u/0/d/12nF9RdN-w2DfufquYDwp-PJB1AYsbbDziEInAaENBJg/edit)
+- [ ] [Design documentation, up to Sept 2020](https://docs.google.com/presentation/d/1gwK4b-R3izYsAh12MVWkFknWDR7URkxTPp1jHU9gULc/edit)
 
-**Done when** :  New team member says so ! Should have had the time to digest info, connect with team and the access needed are granted, **and [audit document](https://docs.google.com/spreadsheets/d/1nqRmotJvNY_ftNwr5R7_tk2HKrveNRBA-R77kNTU-FM/edit#gid=0) has been updated**.
+## Learn team process
+- [ ] [How to use Zenhub](https://docs.google.com/presentation/u/0/d/1_sPItXDa3q1ixZd3ZVQVQJWWmKn_mgInFMiurnD-ya8/edit)
+- [ ] [Story point estimation](https://drive.google.com/file/d/1d2offGEmop4dMZFrCQGPKPqMNoK7n9NW/view)
+- [ ] [Team rituals](https://docs.google.com/document/d/1h4kTrfP257J4QVeSPNAr2-tgV-SM2BbyBbNTUb3WPdc/edit)
+- [ ] [Types of documentation](https://docs.google.com/spreadsheets/d/1sondKNE4bBRN8D6CEYx138vpHVojfOsO71mAz6qGwqs/edit#gid=0)
 
-#  Read and discover **(the following are suggested in order of priority. Reading all of it is not mandatory)**
+## Learn about stakeholders
+- [ ] [Platform business unit - Strategy 2022-23](https://docs.google.com/presentation/u/0/d/1TWAi89DsG696pT4bmZV54Mjn_KP0TzP9R6_elkUDbqU/edit)
 
-- [ ] [Overview of Notify product](https://docs.google.com/document/d/1naerXecHaMR6-udeYiny5a_84CvIKwITAlwR8ZwsIZY/edit#) 
-- [ ] [Notify Scavenger Hunt](https://docs.google.com/document/d/1MmKBZqMNpsLU0TlURGCyOlByZHGfZccteuYdGZ9amwM/edit#heading=h.sxqbcn4k0lmo)🕵️‍♀️
-- [ ] [Directory of links](https://docs.google.com/document/d/15dk3fz6BwX0_ZwHyCXMDYIWCszXSoFarqydKQNeeyx0/edit)
```

Comment:
As we are removing this document, there are links that it contained worth sharing.

For Devs:

* [Technical Playbook](https://cds-snc.github.io/technical-playbook-manuel-technique/)
* [Code review guidelines](https://github.com/cds-snc/cra-claim-tax-benefits/blob/main/docs/CODE-REVIEW-GUIDELINES.md)

For all:

* [Service-Level Agreement for GC Notify](https://docs.google.com/document/d/1JAuWT6XwYUFRetSFffcfCJOuUryW1D8yPeAa4NkWWgs/edit#heading=h.pbh32eniyzdc)

Reply:
Nice! I added the two devs docs to this onboarding doc (linked in the Deeper Dive section) and added the SLA as a deeper dive item for anyone.

Reply:
It took me a second to figure out how to do it in an existing PR :D .... You should see it now! .. here?

Reply:
I see the SLA change but not the technical playbook and code review guidelines! 😵